### DISCUSSION TITLE
Can i upload copyrighted images to my private github repo's?

### DISCUSSION THREAD

**Discussion by SystemPreferences** [community/community#36559] [Other Feature Feedback, Questions, & Ideas] [answered]
I've looked around a bit but can't find a solid answer to this question, if anybody knows sharing would be greatly appreciated.

**Comment by Undertone0809** [comment]
You'd better not. Actually, it's nothing if you don't use the images to the business. If you use it for public welfare, I think it is beyond reproach.

**Comment by radar-blip** [comment, marked as answer]
As long as you don't claim them as your own (that could lead to trouble). 
Giving credit to whoever they belong to would be nice. 
And asking permission to use them would be best.

**Reply by SystemPreferences** [reply]
Thank you for your input.

**Reply by radar-blip** [reply]
Not a problem.

**Comment by ossanna16** [comment]
Hey there @SystemPreferences and thank you for asking a good question! I'm glad you found our community 🙂
